data/brands/amenity/bank.json

```diff
@@ -20,6 +20,19 @@
     }
   },
   "items": [
+      {
+      "displayName": "Banca Popolare di Cividale",
+      "locationSet": {"include": ["it"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "Banca Popolare di Cividale",
+        "brand:wikidata": "Q15639942",
+        "brand:wikipedia": "it:Banca Popolare di Cividale",
+        "name": "Banca Popolare di Cividale",
+        "matchNames": ["CiviBank", "Civi Bank"],
```

Comment:
Hello @alessandrocarmeli,
thanks for your contribution. Could you please move the matchNames out of the tags array? They should sit o the same level as displayName, locationSet, tags etc.

Thank you!